### ISSUE TITLE
Flowframes Installer Stuck at Installing Pytorch files...

### ISSUE THREAD

**Issue by sabswelt** [n00mkrad/flowframes#221]
I have been trying to install Flowframes V1.36.0. The installer is stucked at installing Pytorch files for hours.

Installing application files...
Download completed.
Extraction completed.
Installing Pytorch files...

How do I resolve this pls?
![FF](https://user-images.githubusercontent.com/54246944/226192411-45fbe4d2-2027-4957-a3b1-9e69323751e7.png)

**Comment by amauryjaphet** [comment]
you can install it manually from this website: https://pytorch.org/ (scroll down until you found _install pytorch_)

**Comment by neek8044** [comment]
While someone can install Pytorch manually from https://pytorch.org/ as *amauryjaphet* commented, to explain why this issue actually happened was because the user did not have Python installed, which also installs pip from where I suppose the program installs Pytorch, torchvision and all the other dependencies. 
The requirements link is not easily visible if you just want to install Flowframes quickly, but in the README file there is actually a link that prompts you here: [PythonDependencies.md](https://github.com/n00mkrad/flowframes/blob/main/PythonDependencies.md). The file includes the steps needed to be taken in order to get everything ready. You will see the guide tells you to get Python 3.8.6, but in reality you do not necessarily have to get 3.8.6 specifically (as mentioned in the guide at the time of writing this). I suggest downloading Python 3.10.x, but you do you. 
I hope someone does a PR to detect if the user has Python installed and display an error/warning, instead of just hanging up on "Installing Pytorch..."

***TLDR**: In order to solve this, someone could either install Pytorch manually as stated in the previous comment, or install Python and let Flowframes installer do the rest automatically.*

**Comment by sabswelt** [comment]
I have got Python installed and was installing Pytorch which did not complete and displayed the error in the image. Any Idea of what else is causing this to fail at this point?
![pytorch](https://user-images.githubusercontent.com/54246944/232416278-701ff0ec-2428-45b1-bc1d-76e1a5c8adbf.png)

**Comment by neek8044** [comment]
> I have got Python installed and was installing Pytorch which did not complete and displayed the error in the image. Any Idea of what else is causing this to fail at this point? ![pytorch](https://user-images.githubusercontent.com/54246944/232416278-701ff0ec-2428-45b1-bc1d-76e1a5c8adbf.png)

Could you provide the whole error message please? Only the top of it is shown in the screenshot.
I also suggest making a new issue regarding your problem since you have problems with pip and not flowframes itself.

**Comment by shibi786313** [comment]
> While someone can install Pytorch manually from https://pytorch.org/ as _amauryjaphet_ commented, to explain why this issue actually happened was because the user did not have Python installed, which also installs pip from where I suppose the program installs Pytorch, torchvision and all the other dependencies. The requirements link is not easily visible if you just want to install Flowframes quickly, but in the README file there is actually a link that prompts you here: [PythonDependencies.md](https://github.com/n00mkrad/flowframes/blob/main/PythonDependencies.md). The file includes the steps needed to be taken in order to get everything ready. You will see the guide tells you to get Python 3.8.6, but in reality you do not necessarily have to get 3.8.6 specifically (as mentioned in the guide at the time of writing this). I suggest downloading Python 3.10.x, but you do you. I hope someone does a PR to detect if the user has Python installed and display an error/warning, instead of just hanging up on "Installing Pytorch..."
> 
> _**TLDR**: In order to solve this, someone could either install Pytorch manually as stated in the previous comment, or install Python and let Flowframes installer do the rest automatically._

bro you are amazing it worked for me thanks a lot

**Comment by Chamzyx4** [comment]
hello i have an issue with flowsframe
